components/aws/sagemaker/tests/integration_tests/component_tests/test_train_component.py

```diff
@@ -78,4 +79,9 @@ def test_trainingjob(
     else:
         assert f"dkr.ecr.{region}.amazonaws.com" in training_image
 
+    error_message = 'Error in fetching CloudWatch logs for SageMaker job'
+    if argo_utils.find_in_logs(workflow_json["metadata"]["name"], error_message):
+        print(error_message)
```

Comment:
Let's print a more useful error. Something that we can trace back to this part of the integration test.

Reply:
Perhaps more of a `"Found the CloudWatch error message in the log output. Check SageMaker to see if the job has failed."`

Reply:
okay